templates/Mixing_Score.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
<p>Does this <code>dist_mat_dir</code>line up with the naming in the other spatial analysis notebooks? It's the same distance, matrix? So it only needs to be computed once?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Yes it is the same matrices and wouldn't need to be computed again if they're already saved to a top level directory!

However, I need to check with @alex-l-kong because the current spatial notebooks create separate directories i.e. even if you generate the distance matrices for neighborhood analysis, it would compute it again for spatial enrichement.
`dist_mat_dir = os.path.join(base_dir, "spatial_analysis/pairwise_spatial_enrichment/dist_mats")`
`dist_mat_dir = os.path.join(base_dir, "spatial_analysis/neighborhood_analysis/dist_mats")`
I had thought we were planning to save them somewhere like `"../data/example_dataset"`